src/main/java/net/minecraftforge/installer/json/Util.java

```diff
@@ -143,4 +146,19 @@ public static String replaceTokens(Map<String, ? extends Supplier<String>> token
 
         return buf.toString();
     }
+
+    public static Object createLogButton() {
+        JButton button = new JButton("Open Log");
+        button.addActionListener(ev -> {
+            File file = new File("installer.log");
+            try {
+                if (file.exists())
+                    Desktop.getDesktop().open(file);
+            } catch (IOException e) {
+                // Handle any errors that may occur during file opening
+                e.printStackTrace();
```

Comment:
Should catch `Exception` instead to account for the other types of errors, such as unsupportedoperationexception and securityexception. Ideally show an error popup ui when this happens too

Reply:
Alright

Reply:
Remember to change this from `IOException e` to `Exception e`